app/src/main/java/org/oppia/android/app/settings/profile/SnackbarController.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.android.app.settings.profile
```

Comment:
This controller should go into the domain module. It can reasonably go under a new "snackbar" subpackage, I think.